README.md

```diff
@@ -28,9 +28,9 @@ The extension _should_ work anywhere VS Code itself and PowerShell Core 7 or hig
 PowerShell Core 6 is end-of-life and so not supported. Our test matrix includes the
 following:
 
-- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.1
-- **macOS 10.15** with PowerShell Core 7.2.1
-- **Ubuntu 20.04** with PowerShell Core 7.2.1
+- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.4
```

Comment:
```suggestion
- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.5
```

(unless intentional)

Reply:
Intentional, I checked CI, and didn't want to lie about what it's running _today_.